test/e2e/support/utils/are-pre-publish-checks-enabled.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * Verify if publish checks are enabled.
```

Comment:
Can we use the form of 3rd person for consistency?

`Verifies ...`

Reply:
Adjusted, thx.